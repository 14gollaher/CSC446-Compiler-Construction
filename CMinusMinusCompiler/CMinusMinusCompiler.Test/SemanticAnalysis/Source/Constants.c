const x1 = 3;
int main()
{
	const x2 = 4.2;
}
const y1 = 4;

/*This will be omitted!!!*/
