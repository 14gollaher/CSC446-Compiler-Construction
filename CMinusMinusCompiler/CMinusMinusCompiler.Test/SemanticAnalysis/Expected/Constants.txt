Lexeme                                 Class                          Depth
---------------------------------------------------------------------------
x2                                     Constant                       2


Lexeme                                 Class                          Depth
---------------------------------------------------------------------------
x1                                     Constant                       1
y1                                     Constant                       1


Completed processing Constants.c
